canvas_modules/common-canvas/src/common-properties/properties-main/properties-main.jsx

```diff
@@ -501,12 +501,14 @@ class PropertiesMain extends React.Component {
 				propertiesTitle = (<TitleEditor
 					labelEditable={formData.labelEditable}
 					help={formData.help}
+					description={formData.description}
 					controller={this.propertiesController}
 					helpClickHandler={this.props.callbacks.helpClickHandler}
 					closeHandler={applyOnBlurEnabled ? this.applyPropertiesEditing.bind(this, true) : null}
 					icon={formData.icon}
 					heading={formData.heading}
 					showHeading={this.props.propertiesConfig.heading}
+					showHeadingDesc={this.props.propertiesConfig.headingDesc}
```

Comment:
I'd probably have a new config item `showHeadingDesc`.  We'll also need documentation updates for this.